src/contracts/libraries/GPv2Transfer.sol

```diff
@@ -45,23 +45,26 @@ library GPv2Transfer {
             "GPv2: cannot transfer native ETH"
         );
 
-        if (transfer.balance == GPv2Order.BALANCE_INTERNAL) {
+        if (transfer.balance == GPv2Order.BALANCE_ERC20) {
```

Comment:
What's the value of having `transferFromAccount` and `transferFromAccount**s**` as two methods? Couldn't the former be implemented via the latter?

Reply:
They have slightly different semantics because of gas optimizations. Specifically, we don't withdraw internal balances in `transferFromAccount`.

Naming is hard :see_no_evil: